prov/rxm/src/rxm_cq.c

```diff
@@ -98,6 +98,25 @@ static int rxm_match_iov(const struct iovec *iov, void **desc,
 	return FI_SUCCESS;
 }
 
+static inline int
+rxm_cq_rewind_multi_recv_buffer(struct rxm_iov *multi_recv_iov, size_t total_len,
+				uint64_t offset)
+{
+	if (multi_recv_iov->count == 1) {
+		multi_recv_iov->iov[0].iov_base =
+			(char *)multi_recv_iov->iov[0].iov_base + offset;
+		multi_recv_iov->iov[0].iov_len = total_len;
+		return 0;
+	} else {
+		struct rxm_iov rxm_iov = *multi_recv_iov;
+		assert(multi_recv_iov->count > 1);
+		return rxm_match_iov(rxm_iov.iov, rxm_iov.desc, rxm_iov.count,	/* prev iovecs */
+				     offset,					/* offset */
+				     total_len,					/* match_len */
+				     multi_recv_iov);				/* match_iov */
+	}
+}
+
```

Comment:
Can you please add some details in the commit message regarding this change? Specifically, brief description of the change, why it is needed or how it optimizes multi-recv.